package.json

```diff
@@ -93,16 +93,17 @@
     "nyc": "5.5.0",
     "postcss-flexbugs-fixes": "3.2.0",
     "postcss-loader": "2.0.8",
+    "puppeteer": "^1.5.0",
+    "react": "16.2.0",
     "react-dev-utils": "^5.0.0",
     "react-dimensions": "git://github.com/AlesJiranek/react-dimensions",
+    "react-dom": "16.2.0",
```

Comment:
`npm i` automatically sorted these when I installed puppeteer, there are no changes to react here.